README.md

```diff
@@ -13,18 +15,26 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
 - Enabling the collection of traces and custom metrics from your Lambda functions.
 - Managing subscriptions from the Datadog Forwarder to your Lambda and non-Lambda log groups.
 
+## AWS CDK V1 vs AWS CDK V2
+Two versions of Datadog CDK Constructs exist, `datadog-cdk-constructs` and `datadog-cdk-constructs-v2`. These two support the use of `AWS CDK v1` and `AWS CDK v2` respectively.
+
+- Please pay attention to the version of AWS CDK you are using (All new users are likely to be using AWS CDK V2, and this is recommended)
+- If you are using AWS CDK v1, please instead install and import `datadog-cdk-constructs` rather than `datadog-cdk-constructs-v2`
+- `datadog-cdk-constructs-v2` REQUIRES Node 14+, while datadog-cdk-constructs-v1 still supports Node 12.
+- Otherwise, the use of the two packages is identical.
+
 ## npm Package Installation:
```

Comment:
For the package installation instructions, I would include both the v1 and v2 options. I suspect some people might skip reading the top section and just install v2.

Reply:
I've re-written this part to explicitly show installation instructions for AWS CDK v1 vs v2